.github/workflows/cleanup.py

```diff
@@ -0,0 +1,62 @@
+import asyncio
+import datetime as dt
+import logging
+import os
+import re
+from typing import Any, Dict
+
+import aiohttp
+
+
+LIST_CONTAINERS_URL = (
+    "https://api.github.com/"
+    f"users/neuro-inc/packages/container/{os.environ['GH_REPO_NAME']}/versions"
+)
+STALE_PERIOD = dt.timedelta(
+    days=float(os.environ.get("STALE_DAYS", 0)),
+    hours=float(os.environ.get("STALE_HOURS", 0)),
+)
+VERSION_REGEX = r"\d{1,2}\.\d{1,2}\.\d{0,2}"
+
+assert STALE_PERIOD
```

Comment:
Assertions don't work in production code, you should use explicit `if xxx: raise yyy` instead

Reply:
you are right, changed in d347954